stories/immersiveModal/immersiveModalExamples.tsx

```diff
@@ -131,10 +131,12 @@ const ImmersiveModalExamples = (): JSX.Element => {
       </Button>
       {withButtons && (
         <ImmersiveModal
-          onClose={(): void => setWithButtons(false)}
+          onClose={() => setWithButtons(false)}
           footerContent={
             <Button.Container>
-              <Button isFullWidth>cta content</Button>
+              <Button isFullWidth onClick={() => setWithButtons(false)}>
```

Comment:
Just makes the storybook example more correct--CTAs should have onClicks that does an action.